test/built-ins/Atomics/xor/bigint/non-shared-bufferdata.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Rick Waldron.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-atomics.xor
+description: >
+  Atomics.xor will operate on TA when TA.buffer is not a SharedArrayBuffer
+includes: [testBigIntTypedArray.js]
+features: [ArrayBuffer, Atomics, BigInt, TypedArray]
+---*/
+testWithBigIntTypedArrayConstructors(TA => {
+  const buffer = new ArrayBuffer(BigInt64Array.BYTES_PER_ELEMENT * 4);
```

Comment:
```suggestion
  const buffer = new ArrayBuffer(TA.BYTES_PER_ELEMENT * 4);
```

Also in all other places where this applies.